master/docs/manual/cfg-www.rst

```diff
@@ -119,10 +119,10 @@ The available classes are described here:
 
     ::
 
-        from buildbot.plugins import util
+        from buildbot.www import auth
         c['www'] = {
             # ...
-            'auth': util.UserPasswordAuth({"homer": "doh!"}),
```

Comment:
"should be" meaning it is and my patch is wrong, or that we should move it? Right now it's in "master/buildbot/www/auth.py"